backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -88,21 +89,28 @@ public LiveOktaRepository(AuthorizationProperties authorizationProperties,
 
         // By default, when creating a user, we give them privileges of a standard user
         String organizationExternalId = org.getExternalId();
-        String groupName = generateGroupName(organizationExternalId, OrganizationRole.getDefault());
+        Set<OrganizationRole> roles = EnumSet.of(OrganizationRole.getDefault(), role);
+        Set<String> groupIds = new HashSet<>();
+        
+        for (OrganizationRole r : roles) {
+            String groupName = generateGroupName(organizationExternalId, r);
 
-        // Okta SDK's way of getting a group by group name
-        GroupList groups = _client.listGroups(groupName, null, null);
-        if (!groups.iterator().hasNext()) {
-            throw new IllegalGraphqlArgumentException("Cannot add Okta user to nonexistent group");
+            // Okta SDK's way of getting a group by group name
+            GroupList groups = _client.listGroups(groupName, null, null);
```

Comment:
It seems like Okta _ought_ to support getting all our relevant groups in one go and then iterating over them, which might improve performance as well. Not a blocker here, but we should probably have a collection of Okta performance issues (like #927 or possibly just expanding that issue) to address as we have time.

Reply:
expanding https://github.com/CDCgov/prime-simplereport/issues/927 accordingly